### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by DebyGrey**
## Bookstore: styling

In this project, I:

- Added every missing JSX element to the project

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -77,17 +72,11 @@ nav ul li a {
   color: #fff;
 }
 
-.book-form input,
-select {
+.book-form input {
```
Comment (victorokoroji) [inline src/App.css]: - [ ] Kindly ensure that your form design matches the provided zeplin design 👍🏽 . For you to be professional, it is essential we always stick to design as much as possible.

zeplin design 👇🏽 
image

your design 👇🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good UI design ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/App.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @DebyGrey 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**